_posts/2025-10-05-update.md

```diff
@@ -44,4 +44,12 @@ slug: 2025-10-05-update
 
 ## Shoutouts
 
-* 
\ No newline at end of file
+* Michelle Nguyen -- A big shout-out to Kubernetes v1.34 :of-wind-and-will-party: Docs team! @Rashan @Urvashi @Arvind Parekh @YuJen Huang(Dylan) @DangerBuff
+  You all consistently went above and beyond—whether updating meeting notes meticulously, tracking down docs, or supporting each other with krel tasks. 
+  Every single person delivered exceptional work without fail. Thanks to you all, our release was smooth, especially from a docs perspective!
+  A special shoutout to @Drew Hagen for helping Docs out during Docs Freeze. You absolutely rock! I'm incredibly proud of what we've accomplished as a team and am extremely grateful for the opportunity to work alongside everyone.
+* Drew Hagen -- I'd like to take a moment to acknowledge @Matteo (away until Jan 2026) for the seriously impressive leadership of a newer release branch management shadow program for the 1.34 release, and all the amazing work putting together strong documentation for branch management!! :extreme-teamwork:
```

Comment:
```suggestion
* Drew Hagen -- I'd like to take a moment to acknowledge @Matteo for the seriously impressive leadership of a newer release branch management shadow program for the 1.34 release, and all the amazing work putting together strong documentation for branch management!!
```
When copying the messages from Slack, let's remove the obvious Slack specific bits such as emojis and statuses set by folks.